modules/material/material.resolver.js

```diff
@@ -16,6 +16,16 @@ const materialQuery = {
       message: MATERIAL_NOT_FOUND,
     };
   },
+  getMaterialColorByCode: async (parent, args) => {
+    const material = await materialService.getMaterialColorByCode(args.code);
+    if (material[0].colors[0]) {
+      return material[0].colors[0];
```

Comment:
Move this logic to service, we should try to remain resolvers as clean as possible

Reply:
Fixed